internal/service/iam/service_specific_credential_test.go

```diff
@@ -47,6 +47,45 @@ func TestAccServiceSpecificCredential_basic(t *testing.T) {
 	})
 }
 
+func TestAccServiceSpecificCredential_multi(t *testing.T) {
+	var cred iam.ServiceSpecificCredentialMetadata
+
+	resourceName := "aws_iam_service_specific_credential.test"
+	resourceName2 := "aws_iam_service_specific_credential.test"
```

Comment:
Should this be "aws_iam_service_specific_credential.test2"

Reply:
🤦  fixed